code/_globalvars/game_modes.dm

```diff
@@ -7,3 +7,5 @@ GLOBAL_DATUM(start_state, /datum/station_state) // Used in round-end report
 
 /// We want reality_smash_tracker to exist only once and be accessible from anywhere.
 GLOBAL_DATUM_INIT(reality_smash_track, /datum/reality_smash_tracker, new)
+
+GLOBAL_VAR(deathmatch_game) // Deathmatch Minigame controller
```

Comment:
```suggestion
GLOBAL_DATUM(deathmatch_game, /datum/deathmatch_controller) // Deathmatch Minigame controller
```
Then you won't need to declare the type of this global datum everytime you've to access it...